lib/log_parser.rb

```diff
@@ -0,0 +1,12 @@
+class LogParser
+
+  def initialize(file_path)
+    @file_path = file_path
+    raise "File #{file_path} wasn't found" unless File.exist?(@file_path)
+  end
+
+  def readFirstLine
```

Comment:
por convensão no ruby declaramos as funções em `snake_case`
```suggestion
  def read_first_line
```